### ISSUE TITLE
Windows version?

### ISSUE THREAD

**Issue by rayman3003** [tihu-nlp/tihu#53]
I can see only the linux version in the release section.

Can we have a compiled **windows** version of tihu, in release section too?

**Comment by b00f** [comment]
I never compiled it for Windows. I really don't know if it is possible or not. Not bad give a try

**Comment by b00f** [comment]
Yes, true. I am sure it can be compiled in Windows. Probably we need some small changes. Feel free to open Pull Requests
